tests/impl.cpp

```diff
@@ -1823,6 +1823,22 @@ result_t test_mm_free(const SSE2NEONTestImpl &impl, uint32_t iter)
     return TEST_UNIMPL;
 }
 
+result_t test_mm_get_flush_zero_mode(const SSE2NEONTestImpl &impl,
+                                     uint32_t iter)
+{
+    int res_flush_zero_on, res_flush_zero_off;
+    _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);
+    res_flush_zero_on = _MM_GET_FLUSH_ZERO_MODE() == _MM_FLUSH_ZERO_ON;
+    _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_OFF);
+    res_flush_zero_off = _MM_GET_FLUSH_ZERO_MODE() == _MM_FLUSH_ZERO_OFF;
+
+    if (res_flush_zero_on && res_flush_zero_off) {
+        return TEST_SUCCESS;
+    } else {
```

Comment:
Remove `else` statement since it is not necessary.

Reply:
I'll rewrite it as `return (res_flush_zero_on && res_flush_zero_off) ? TEST_SUCCESS : TEST_FAIL;`

Reply:
Done.